unity-renderer/Assets/Rendering/Outliner/OutlineMaskFeature.cs

```diff
@@ -0,0 +1,224 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+public class OutlineMaskFeature : ScriptableRendererFeature
+{
+    private class OutlinerRenderPass : ScriptableRenderPass
+    {
+        private const int DEPTH_BUFFER_BITS = 32;
+        private const string PROFILER_TAG = "Outliner Mask Pass";
+        private const bool USE_BASE_MATERIAL = false; // Use the material in the renderer and look for an "Outliner" pass
+
+        private static readonly int BONE_MATRICES = Shader.PropertyToID("_Matrices");
+        private static readonly int BIND_POSES = Shader.PropertyToID("_BindPoses");
+        private static readonly int RENDERER_WORLD_INVERSE = Shader.PropertyToID("_WorldInverse");
+        private static readonly int AVATAR_MAP1 = Shader.PropertyToID("_AvatarMap1");
+        private static readonly int AVATAR_MAP2 = Shader.PropertyToID("_AvatarMap2");
+        private static readonly int AVATAR_MAP3 = Shader.PropertyToID("_AvatarMap3");
+        private static readonly int AVATAR_MAP4 = Shader.PropertyToID("_AvatarMap4");
+        private static readonly int AVATAR_MAP5 = Shader.PropertyToID("_AvatarMap5");
+        private static readonly int AVATAR_MAP6 = Shader.PropertyToID("_AvatarMap6");
+        private static readonly int AVATAR_MAP7 = Shader.PropertyToID("_AvatarMap7");
+        private static readonly int AVATAR_MAP8 = Shader.PropertyToID("_AvatarMap8");
+        private static readonly int AVATAR_MAP9 = Shader.PropertyToID("_AvatarMap9");
+        private static readonly int AVATAR_MAP10 = Shader.PropertyToID("_AvatarMap10");
+        private static readonly int AVATAR_MAP11 = Shader.PropertyToID("_AvatarMap11");
+        private static readonly int AVATAR_MAP12 = Shader.PropertyToID("_AvatarMap12");
+        private static readonly int _Height = Shader.PropertyToID("_Height");
```

Comment:
Rename to `Height`?

Reply:
Removed, it's no longer used because we are not doing the vertical fade in the en d.